setup.py

```diff
@@ -69,14 +83,81 @@ def run(self):
           os.system('cp third_party/gfootball_engine/_gameplayfootball.so ' +
                     dest_dir)):
         raise OSError('Google Research Football compilation failed')
-    super(CustomBuild, self).run()
 
+  def run_windows(self):
+    try:
+      download_engine = int(os.environ.get('DOWNLOAD_ENGINE', '0'))
+    except ValueError:
+      raise ValueError('Could not parse COMPILE_ENGINE environment '
```

Comment:
I used to have COMPILE_ENGINE variable, but changed it to DOWNLOAD_ENGINE. I'll fix it in the next commit.